web3.md

```diff
@@ -293,5 +293,49 @@ module WEB3
     rule <k> #evm_increaseTime => #sendResponse( "result" : Int2String(TS +Int DATA ) ) ... </k>
          <params> [ DATA:Int, .JSONList ] </params>
          <timestamp> ( TS:Int => ( TS +Int DATA ) ) </timestamp>
+```
+
+# eth_estimateGas
+
+```k
+    syntax KItem ::= "#eth_estimateGas"
+ // -----------------------------------
+    rule <k> #halt ~> #eth_estimateGas => #sendResponse( "result": #unparseQuantity( GASUSED ) ) ... </k>
+         <gasUsed> GASUSED </gasUsed>
+
+    rule <k> (#eth_estimateGas => #call ACCTFROM ACCTTO ACCTTO VALUE VALUE DATA false ~> #eth_estimateGas) ... </k>
+         <txPending> ListItem(MSGID) ... </txPending>
+         <origin> ACCTFROM </origin>
+         <message>
+           <msgID> MSGID  </msgID>
+           <to>    ACCTTO </to>
+           <value> VALUE  </value>
+           <data>  DATA   </data>
+           ...
+         </message>
+
+    syntax KItem ::= "loadTX" JSON
+ // ------------------------------
+    rule <k> loadTX J => . ... </k>
```

Comment:
this doesn't seem to be being invoked? also, you are going to need to have some kind of checkTx step that checks that the parameters actually are strings everywhere you call #getString, otherwise it will get stuck and crash the client.